src/components/form/templates/object-field.ts

```diff
@@ -35,13 +35,26 @@ function renderCollapsibleField(props: LimeObjectFieldTemplateProps) {
         'limel-collapsible-section',
         {
             header: props.title,
+            id: getSchemaObjectPropertyPath(
+                props.formContext.schema,
+                props.idSchema
+            ),
             'is-open': defaultOpen,
         },
         renderDescription(props.description),
         renderProperties(props.properties, props.schema)
     );
 }
 
+function getSchemaObjectPropertyPath(schema: any, subSchema: LimeJSONSchema) {
+    const rootStart = 2;
```

Comment:
`rootStart` doesn't really make the number any less magic 😄

How about `numberOfCharactersToRemoveFromBeginningOfString`?

Reply:
Yeah, I like that 👍